Tests/MetaTests/Extensions/Snapshotting+Codegen.swift

```diff
@@ -15,4 +16,14 @@ extension Snapshotting where Value == URL, Format == String {
     }
     
     static let copyMethod: Snapshotting<URL, String> = .generatedCode(CopyVisitor())
+    
+    static func copyGeneration<D: CodegenDependencies>(for environment: D) -> Snapshotting<URL, String> {
```

Comment:
IMO both methods are the same, one working with `visitor` and the other using `environment` but should be named `generatedCode`, aren't they?

Reply:
They are not the same, the one for the visitor only does the check for a single file generation and the other one receives a directory and recursively checks and generates all files. Naming these two different things is a bit complex, but I am open to renaming it to clarify.